cmake/tests.cmake

```diff
@@ -0,0 +1,75 @@
+include(googletest)
```

Comment:
I think we should just have a single target `tester` which includes all `_test.cc` files as sources. Google Test allows you to specify test filters, so one can still run a subset of tests. Something like this:

```
# Collect test sources
file(GLOB TEST_CC "*_test.cc")

add_executable(tester ${TEST_CC})
target_link_libraries(tester ...)
```

Reply:
Done.

Reply:
I tried doing that, but was running into linker issues where functions from `xorfilter` get defined twice...